borg/testsuite/archiver.py

```diff
@@ -669,6 +669,27 @@ def test_create_dry_run(self):
         manifest, key = Manifest.load(repository)
         self.assert_equal(len(manifest.archives), 0)
 
+    def test_create_topical(self):
+        self.create_regular_file('file1', size=1024 * 80)
+        self.cmd('init', self.repository_location)
+        # no listing by default
+        output = self.cmd('create', self.repository_location + '::test', 'input')
+        self.assert_not_in('file1', output)
+        # shouldn't be listed even if unchanged
+        output = self.cmd('create', self.repository_location + '::test0', 'input')
+        self.assert_not_in('file1', output)
+        # should list the file as unchanged
+        #output = self.cmd('create', '--unchanged', self.repository_location + '::test1', 'input')
+        #self.assert_in('file1', output)
+        # should *not* list the file as changed
+        #output = self.cmd('create', '--changed', self.repository_location + '::test2', 'input')
+        #self.assert_not_in('file1', output)
+        # change the file
+        self.create_regular_file('file1', size=1024 * 100)
+        # should list the file as changed
+        output = self.cmd('create', '--changed', self.repository_location + '::test3', 'input')
+        self.assert_in('file1', output)
```

Comment:
oh, and if you meant to tell me to RTFM *this* section, i already did: http://borgbackup.readthedocs.org/en/stable/usage.html#item-flags

it doesn't explain with two runs of `create` would produce `A` instead of `U` with the same dataset.